api/lib/config.js

```diff
@@ -28,14 +32,18 @@ var configPath = nconf.get('config')
 || process.env['TEST_CONFIG']
 || path.join(process.cwd(), 'config.json');
 
+if (!fs.existsSync(configPath)) {
```

Comment:
I don't think that's true. When I tried to run "npm test" without a config.json file, it crashed with a massive stack trace and a message like "servers must be an array". That is why I made this pull request - because it wasn't immediately obvious what was wrong from that exception message. If it's desired to be able to run the tests without the config file, then I think there is a bug somewhere.